pkgs/development/tools/xcbuild/toolchain.nix

```diff
@@ -58,28 +59,30 @@ stdenv.mkDerivation {
     ln -s ${llvm}/bin/llvm-otool
     ln -s ${llvm}/bin/llvm-profdata
     ln -s ${llvm}/bin/llvm-size
- '';
+
+    ln -s ${yacc}/bin/yacc
+    ln -s ${yacc}/bin/bison
+    ln -s ${flex}/bin/flex
+    ln -s ${flex}/bin/flex++
+    ln -s flex lex
+    ln -s ${m4}/bin/m4
+  '';
 }
 
 # other commands in /bin/
 #   asa
-#   bison
 #   c89
 #   c99
```

Comment:
I think `c89` and `c99` are entry points for `clang`, but I don't know what produces them.